app/controllers/jitsi_calls_controller.rb

```diff
@@ -2,22 +2,119 @@ class JitsiCallsController < ApplicationController
   before_action :authenticate_user!
 
   def create
+    return unless check_user_availability
+
     @jitsi_call = JitsiCall.new(room_name: SecureRandom.uuid)
-    if @jitsi_call.save && User.exists?(call_params[:participant_id])
+    if @jitsi_call.save
       assign_participants
+      notify_participants
+      notify_initiator
     else
       redirect_to root_path, alert: I18n.t('denial.resource_creation', resource: JitsiCall)
     end
   end
 
+  def accept
+    call = JitsiCall.find(params[:id])
+    return unless call.invitation(current_user).state == MeetingInvitation.state_requested
+
+    change_call_state MeetingInvitation.state_accepted
+
+    data = { action: :start_call, url: call.url, popup_text: I18n.t('call.starting') }
+    send_notification(current_user, **data)
+    send_notification(call.initiator, **data)
+  end
+
+  def reject
+    call = JitsiCall.find(params[:id])
+    return unless call.invitation(current_user).state == MeetingInvitation.state_requested
+
+    change_call_state MeetingInvitation.state_rejected
+
+    send_notification(
+      call.initiator,
+      action: :simple_message,
+      popup_text: I18n.t('call.rejected', guest: current_user.display_name),
+      dismiss: I18n.t('confirmation.dismiss')
+    )
+  end
+
+  # The initiator cancels a outgoing call
+  def abort
+    call = JitsiCall.find(params[:id])
+    unless call.initiator == current_user && (call.invitation(current_user).state == MeetingInvitation.state_accepted)
```

Comment:
This should check whether the meeting has started (was already accepted by a guest)